### DISCUSSION TITLE
Can you make a video tutorial building it on youtube

### DISCUSSION THREAD

**Discussion by Am0stafa** [DustinBrett/daedalOS#415] [General]
I would be grate since me and I believe a lot of people has hard time understanding how it is build

**Comment by Am0stafa** [comment]
I know it took years to build but you can make a video explaining how to build it with out tests and other stuff

**Comment by DustinBrett** [comment]
Thanks for the suggestion! I actually did stream live 52 weeks in a row for 2-3 hours each time in 2021 when I started the project. And I have done nearly 30 other videos also about the project, some with a lot of detail about how I am making it.

https://www.youtube.com/playlist?list=PLM88opVjBuU7xSRoHhs3hZBz3JmHHBMMN

I do plan to do much more and also again do live streams when I eventually rewrite daedalOS from scratch in vanilla JS for v3.
